src/service-worker.ts

```diff
@@ -46,7 +46,9 @@ self.addEventListener("fetch", (event: any /* ExtendableEvent */) => {
 
 	// always serve static files and bundler-generated assets from cache
 	if (url.host === self.location.host && cached.has(url.pathname)) {
-		event.respondWith(caches.match(event.request));
+		caches.match(event.request).then((match): void => {
+			if (match) event.respondWith(match);
```

Comment:
Are there side effects to letting the `event` never be responded to if there's no match?

By the way, this "solves" https://github.com/sveltejs/sapper/issues/760#issuecomment-632974392

Reply:
I don't know. I haven't used service workers much. But I couldn't find any examples online that did anything else